### ISSUE TITLE
POC angular shell/navigation UI for new SORMAS features

### ISSUE THREAD

**Issue by MartinWahnschaffe** [hzi-braunschweig/SORMAS-Project#11630]
### Feature Description
As the long-term goal is to move from Vaadin to Angular, we need a foundation that allows us to:

1. Move existing SORMAS features (e.g. mSERS, Case Surveillance, etc.) from Vaadin to Angular
2. Build the UI for new features only in Angular and no longer in Vaadin

The https://github.com/hzi-braunschweig/SORMAS-Angular project provies a lot of the core functionality of SORMAS. Unfortunately none of the features is complete in a way that would allow it to completely replace the existing SORMAS Vaadin UI.

### Proposed Change
The idea is to run the Vaadin UI and Angular UI in parallel. We will not embedded one UI into the other. Instead, we will use the different views of SORMAS as the separation between the two frontends. In other words, we will switch between both frontends using links.
E.g.` .../sormas-ui/#!cases` would be provided by the Sormas Vaadin UI, while `.../sormas-web/about` would be provided by the Sormas Angular UI.

Goal:

* sormas-web module in the SORMAS-Project repository with all the basic dependencies that can be copied from SORMAS-Angular OR keep the angular app in the existing repository? The second option would only be a temporary solution. Long-term the whole vertical slice from backend to frontend should be in one repository.
* Navigation menu that looks like the one of the current Vaadin application and needs to be configured based on which features are available (as configured in the backend) and thus have an entry in the navigation menu.
* An easy way to provide URL of each view, so the correct frontend is used. This either needs to be kept in sync for both frontends or use a shared logic in the backend (preferred).
* Pick one view that will be provided by the angular app from now on (e.g. about page)
* Make sure use login works seamless.

The scope of this issue it to create a POC for this and needed follow-up issues.

Nice to have / future:

* Some views may be available in both UIs - for those it would be nice to allow the user to switch between both.

### Additional Information

Ideally we would be able to use this so https://github.com/hzi-braunschweig/SORMAS-Project/issues/11533 can be implemented in the new frontend.

**Comment by MartinWahnschaffe** [comment]
I have created the #11666 epic and the most important ticket for the integration which is #11665. I still need to discuss this with someone and probably create more detailed follow-up issues, so I don't consider this POC done for now.

**Comment by MartinWahnschaffe** [comment]
Snippet to change URL of vaadin view for navigation:

```
navigator.addViewChangeListener((viewChangeEvent) -> {
	if (viewChangeEvent.getViewName() == AboutView.VIEW_NAME) {
		Page.getCurrent().setLocation("http://localhost:4200/about");
		return false;
	}
	return true;
});
```

**Comment by StefanKock** [comment]
Open: Link to the draft that was produced as part of this ticket
